src/main/java/com/traveltime/benchmarks/BenchmarkSetup.java

```diff
@@ -39,6 +41,21 @@
             throw new RuntimeException("Could not start benchmark. The following required environment variables were not set: " + nulls);
         }
 
+        val destinationCountStringOrNull = System.getenv("DESTINATION_COUNTS");
+
+        if (destinationCountStringOrNull == null) {
+            destinationCounts = new String[]{"10000", "20000", "40000", "100000"};
+        } else {
+            try {
+                destinationCounts = destinationCountStringOrNull.split(",");
+                Arrays.stream(destinationCounts).forEach(Integer::valueOf);
+            } catch (Exception e) {
+                throw new RuntimeException("Could not start benchmark. Unable to parse environment variable DESTINATION_COUNTS " +
```

Comment:
You can add the original exception as a second parameter (`cause`) of the constructor for an even more precise error message.

Reply:
Sure, though I figured this is all the info anybody could possibly need. I guess I could also assert that `Integer::valueOf` returns a positive number.

Reply:
And maybe one that is smaller than 5M? Also, the original exception will print the exact string that failed the parse, so it would be easier to see if the user mistyped something.